src/box/lua/console.lua

```diff
@@ -294,6 +298,15 @@ local function set_output(_storage, value)
     return true
 end
 
+local function set_continuation(storage, value)
+    if (value == nil) then
+        storage.multiline_delim = ''
+    else
+        storage.multiline_delim = value
+    end
+    return true
+end
```

Comment:
I see no point in ability to set anything except a backslash. Now about just `\set continuation false|true`?

Reply:
We agreed to have:
`\set continuation on|off`

Reply:
See comment above: I don't understand why the implementation still stores the string (backslash or ''), calculates its length and so on. Let's just store whether the continuation is enabled.